config/template_config.toml

```
@@ -11,9 +12,9 @@
 # he - Hebrew, by https://github.com/netanelkoli
 # es_mx - Spanish (Mexican), by https://github.com/mastersuv
 # The lanugages that messages can be displayed in
-enabled_languages = ["it", "en", "uk", "ru", "zh_cn", "he", "es_mx"]
+enabled_languages = ["br","it", "en", "uk", "ru", "zh_cn", "he", "es_mx"]
 # The default language to be set for users whose language cannot be autodetected or whose language is not enabled
-default_language = "it"
+default_language = "br"
```

```suggestion
default_language = "it"
```

Please don't change the default language :)